.github/workflows/coverage.yml

```diff
@@ -0,0 +1,55 @@
+name: Coverage
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+env:
+  RUSTFLAGS: "-D warnings"
+
+jobs:
+  coverage:
+    runs-on: ${{ matrix.platform }}
+    name: "${{ matrix.example }} - ${{ matrix.platform }}"
+    strategy:
+      matrix:
+        platform: [macos-latest]
+        toolchain: [stable]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Homebrew
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+      - name: Install LLVM
+        run: brew install llvm
```

Comment:
It will take too much time on every CI run. Can we use Linux as a host? It will be much faster to install it there.

Reply:
in this repo: https://github.com/near/near-sdk-rs/actions/runs/10295407665/job/28495033969?pr=1236
in my fork: https://github.com/jrmncos/near-sdk-rs/actions/runs/10295407683/job/28495034208?pr=1

it's takes ~8 approximately. Let's try with Linux to reduce it

Reply:
hey @frol I tried with ubuntu, and the longer step, that is 'Generate code coverage' it takes 7m:48seg https://github.com/near/near-sdk-rs/actions/runs/10360531522/job/28679132518
and with mac os 5m 36s
https://github.com/near/near-sdk-rs/actions/runs/10360643219/job/28679480453?pr=1236\

Even with the installation of Brew the total time of macos is still less. Also, the longer step in the GitHub actions pipelines it's Test core and it takes > 8m (this is the total time of coverage)

I tried using    - uses: Swatinem/rust-cache@v1
 as cache but has not improved the total time